allvers/2007-07-16-GPL-2.0-from-gnu.org.txt.notes

```diff
@@ -0,0 +1,9 @@
+Date: 2007-07-16
+Links:
+    - http://web.archive.org/web/20070716031727/http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt
+
+Notes: The mostly final version of the GPL 2 aka, the "graveyard" version.
+ Shortly after the publication of the GPL 3.0, the good ole GPL 2.0
+ was retired to the old licenses cemetery. And along the GPL 2
+ received a cosmetic touchup with pucntuation and and formatting fixes
```

Comment:
This sentence could also use some cosmetic touchups ;-)

Reply:
Updated in 0fec4597c3c06e5477a6deb31db8b2e6508a6a65